gdcdictionary/schemas/subject.yaml

```diff
@@ -463,35 +206,13 @@ properties:
         term_id: C68615
         term_version: 20.03e (Release date:2020-03-31)
 
-  age_decade:
-    description: >
-      the age of the patient in decades
-    type: string
-
   isolated:
```

Comment:
This property was recommended for removal and will be captured instead by the isolation_status property.

Reply:
done